app/keepers/keepers.go

```diff
@@ -296,14 +296,23 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		appKeepers.SlashingKeeper,
 	)
 
+	gammKeeper := gammkeeper.NewKeeper(
+		appCodec, appKeepers.keys[gammtypes.StoreKey],
+		appKeepers.GetSubspace(gammtypes.ModuleName),
+		appKeepers.AccountKeeper,
+		// TODO: Add a mintcoins restriction
+		appKeepers.BankKeeper, appKeepers.DistrKeeper, appKeepers.ConcentratedLiquidityKeeper)
```

Comment:
Note to reviewer: we can't just move up the gammKeeper initialization from below since that one needs clKeeper to be properly set up

Reply:
Can we just duplicate a keeper initialization like this? It feels really odd / wrong

Reply:
Forgot to update this comment -- I implemented it correctly in the next PR against this one. Will upstream those changes here.